components/gallery/GalleryItemAction/GalleryItemAction.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <div>
     <!-- price -->
-    <GalleryItemPriceBuy v-if="!isOwner" />
+    <GalleryItemPriceBuy
+      v-if="!isOwner && nft?.id && nft.currentOwner"
```

Comment:
could you explain this change? `nft.currentOwner`?

Reply:
I don't think I wrote it.
I think it came from a merge with main

Reply:
it was bad undefined props handling.
resolved